### DISCUSSION TITLE
Open Dialog example behaving weirdly https://alpinejs.dev/plugins/focus

### DISCUSSION THREAD

**Discussion by khanakia** [alpinejs/alpine#3157] [7. Bugs (archived)]
When i click the Open Dialog button in the x-trap section and the scroll down to the nested section and click the Open Dialog button there it scrolls to the already opened dialog into the above section.

Please check the video.

https://user-images.githubusercontent.com/2174170/190117955-fbba73a8-0e70-4f43-88b3-4957662aacbd.mov

**Comment by SimoTod** [comment]
The alpine site is just showcasing the usage. Can you post an example to reproduce the issue on a normal page? Bear in mind that, when x-trap is used in a real page, you can't interact with other element while the focus is trapped.

**Reply by khanakia** [reply]
I am aware that it's just a showcase. But the function should work there correctly also which is not the case at the moment. So I was pointing out the issue we should fix issue into the demo site as well.

**Reply by SimoTod** [reply]
In that case, you need the maintainer (I checked and it also happen on normal pages). 
The focus plugin should trap the focus so this could be the expected behavior.
It uses a different library as a dependency (focus-trap) so it could also be a bug for that repo.

**Reply by khanakia** [reply]
Do you have any idea what framework alpinejs uses for documentation site?

**Reply by SimoTod** [reply]
I think it's hosted on github pages and uses jekyll
